frontend/src/components/EmployeeCards/EmployeeCards.tsx

```diff
@@ -103,25 +103,26 @@ const EmployeeCard = ({
     </Link>
   );
 };
+
 const EmployeeCards = () => {
   const { admins, drivers } = useEmployees();
 
+  const allDrivers = [...admins, ...drivers];
+  allDrivers.sort((a: Driver | Admin, b: Driver | Admin) => {
+    if (a.firstName < b.firstName) { return -1; }
+    if (a.firstName > b.firstName) { return 1; }
```

Comment:
maybe do it by full name instead, and move sorting into the EmployeeContext so its consistent throughout the site

Reply:
1. maybe do it by full name instead
I don't think there's a fullName attribute for employee and driver objects. Could you clarify what you mean?

2. move sorting into the EmployeeContext so its consistent throughout the site
I added the alphabetically sorting logic to EmployeeContext, but since EmployeeCards merges admins and drivers into a single allDrivers array, I kept the allDrivers.sort to sort the merged array alphabetically

Reply:
fullName would be ``${a.firstName} ${a.lastName}``

Reply:
wouldn't `${a.firstName} ${a.lastName}` just sort by `a.firstName` ?

Reply:
if there are two people with the same first names, then we'd need to look at the last name